libsolidity/ast/Types.cpp

```diff
@@ -3435,7 +3435,7 @@ MemberList::MemberMap TypeType::nativeMembers(ContractDefinition const* _current
 				members.emplace_back(declaration->name(), declaration->type(), declaration);
 			else if (
 				(contract.isLibrary() && declaration->isVisibleAsLibraryMember()) ||
-				declaration->isVisibleViaContractName()
+					declaration->isVisibleViaContractTypeAccess()
```

Comment:
Why indent?

Reply:
Because it's what my IDE did, when I had it rename this automatically and I didn't notice the indent in the diff - I guess :-). I'll fix while squashing.